lib/OnUserTypingHandler.ts

```diff
@@ -7,6 +7,13 @@ import { ErrorLogs } from '../enum/ErrorLogs';
 import { getRoomAssoc, retrievePersistentData } from '../helperFunctions/PersistenceHelpers';
 import { chasitorSneakPeak, chasitorTyping } from '../helperFunctions/SalesforceAPIHelpers';
 import { getAppSettingValue } from '../lib/Settings';
+import { resetAppTimeout } from '../helperFunctions/TimeoutHelper';
+
+let allowTimeoutReset = true;
+
+let userStoppedTypingTimeout: NodeJS.Timeout | null = null;
```

Comment:
@csmith14 Were you able to test these changes?

As far as I know, `setTimeout` does not work in the RC Apps.

And also using `allowTimeoutReset`, and `userStoppedTypingTimeout` as a condition won't work here because in the real world this function will be called for multiple chat/user. So callback for one chat will override the flag for other chats.